### DISCUSSION TITLE
.CDA files

### DISCUSSION THREAD

**Discussion by sigmun65** [enzo1982/freac#479] [Help & Support]
HI
I can't import .cda files from my external cd reader connected to my PC.
Simply when I try to open one of the .cda tracks the fre:ac tells me that it is an unsupported format.

What is the problem?

Mario

**Comment by enzo1982** [comment]
.cda files do not contain any audio. They are just links to an audio CD track on the currently loaded disc. Thus fre:ac can open a .cda file only if the corresponding disc is still inserted.

If the disc is inserted, but fre:ac still cannot open the .cda files (and also no button to add CD tracks appears in fre:ac's toolbar), please check your settings for _[Controlled folder access](https://support.microsoft.com/en-us/windows/allow-an-app-to-access-controlled-folders-b5b6627a-b008-2ca2-7931-7e51e912b034)_. If _Controlled folder access_ is enabled, an exception needs to be added for fre:ac in order for it to be able to access CD drives.
